tomcat/src/main/java/org/apache/http/HttpRequest.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.http;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.exception.InvalidRequestException;
+
+public class HttpRequest {
+
+    private final HttpMethod method;
+    private final String uri;
+    private final Map<String, String> queryString;
+    private final HttpVersion version;
+
+    public HttpRequest(List<String> message) {
+        if (message.size() <= 0) {
```

Comment:
```suggestion
        if (message.empty()) {
```
를 쓰는 편이 더 가독성이 좋을 것 같습니다.
아니면 아래 size() < 3 과 통일하기 위한 것 인가요?

Reply:
엇! 놓친 포인트였는데 좋은 의견 감사합니다! 바로 수정할게요!